packages/venia-concept/src/components/ForgotPassword/forgotPassword.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classify from 'src/classify';
+import ForgotPasswordForm from './ForgotPasswordForm';
+import FormSubmissionSuccessful from './FormSubmissionSuccessful';
+import defaultClasses from './forgotPassword.css';
+import { resetPasswordRequest } from './api';
+
+class ForgotPassword extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            instructions: PropTypes.string
+        }),
+        initialValues: PropTypes.shape({
+            email: PropTypes.string
+        }),
+        onClose: PropTypes.func
+    };
+
+    state = {
+        email: '',
+        submitSucceeded: false
+    };
+
+    handleFormSubmit = async ({ email }) => {
+        await resetPasswordRequest({ email });
+
+        this.setState({
+            email,
+            submitSucceeded: true
+        });
+    };
+
+    handleContinue = () => {
+        this.setState({
+            email: '',
+            submitSucceeded: false
+        });
+
+        this.props.onClose();
```

Comment:
Since we're calling `this.props.onClose` directly, let's add a `.isRequired` on it in `propTypes`.

Reply:
Yeah, I agree